hebiros_advanced_examples/launch/rviz/x_demo_rviz.launch

```diff
@@ -1,13 +1,13 @@
 <launch>
   <param name="robot_description" command="cat $(find hebiros_description)/urdf/x_demo.urdf"/>
 
-  <node name="hebiros_node" pkg="hebiros" type="hebiros_node" respawn="false" output="screen"/>
+  <node name="hebiros_node" pkg="hebiros" type="hebiros_node" args="-use_gazebo true" respawn="false" output="screen"/>
```

Comment:
Should we instead make a separate launch file for the x demo with gazebo vs. x demo on hardware?